pkg/loader/compose/compose.go

```diff
@@ -374,7 +374,10 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 		serviceConfig.Tty = composeServiceConfig.Tty
 		serviceConfig.MemLimit = composeServiceConfig.MemLimit
 		serviceConfig.TmpFs = composeServiceConfig.Tmpfs
-		komposeObject.ServiceConfigs[name] = serviceConfig
+		komposeObject.ServiceConfigs[normalizeServiceNames(name)] = serviceConfig
+		if !reflect.DeepEqual(komposeObject.ServiceConfigs[normalizeServiceNames(name)], komposeObject.ServiceConfigs[name]) {
+			log.Infof("Service name in docker-compose has been changed from %c%s%c to %c%s%c", '"', name, '"', '"', normalizeServiceNames(name), '"')
```

Comment:
if you want to print `"` in the output then you can use format specifier `%q` and get the same result, so you don't need to do the hack currently done!